kivy/uix/effectwidget.py

```diff
@@ -683,17 +683,11 @@ def __init__(self, **kwargs):
     def _refresh_background_color(self, *args):
         self._background_color.rgba = self.background_color
 
-    def _update_glsl(self, *largs):
-        '''(internal) Passes new time and resolution uniform
-        variables to the shader.
-        '''
-        time = Clock.get_boottime()
-        resolution = [float(size) for size in self.size]
-        self.canvas['time'] = time
-        self.canvas['resolution'] = resolution
+    def _propagate_updates(self, *largs):
+        """Propagate updates from widgets."""
+        del largs
```

Comment:
Why `del largs`?

Reply:
"Callback" requires those largs but we do not use them. This is to prevent any linter from reporting unused parameters (not sure if those are active for kivy currents).

Reply:
I see, could you add an `# unused` comment to clarify what's going on? It would also be good to rename `largs` to `_`, which will actually keep pylint happy on its own since that's a convention it expects for unimportant arguments.